rpcs3/rpcs3qt/richtext_item_delegate.h

```diff
@@ -38,12 +38,7 @@ class richtext_item_delegate : public QStyledItemDelegate
 
 		// Adjust our painter parameters with some magic that looks good.
 		// This is necessary so that we don't draw on top of the optional widget.
-		// If you're not happy with this code don't hesitate to contact Megamouse
```

Comment:
Why are you removing this 🤣

Reply:
Because it was added on the same commit I'm undoing